content/docs/dvclive/how-it-works.md

```diff
@@ -71,71 +71,102 @@ model.pt
 model.pt.dvc
 ```
 
-## Track the results
+## Git and DVC integration
 
-DVCLive expects each run to be tracked by Git, so it will save each run to the
-same path and overwrite the results each time. Include
+DVCLive differs from some other experiment trackers by relying on Git and DVC
+for tracking instead of a central database. This provides a closer connection to
```

Comment:
quick thought: I guess it's somewhat similar to Tensorboard btw (no Git, but also not central database)